config/ra2.ini

```diff
@@ -1,256 +1,258 @@
-25=Play Sound,Number,Plays a sound from SOUND.ini. The parameter is the sound number in [SoundList] - 1.
-26=Play Movie,Number,Plays a movie from [Movies] in ART.ini. The parameter is the movie number - 1.
-27=Play Theme,Number,Plays a theme from THEME.ini. The parameter is the theme number in [Themes] - 1.
-28=Reduce Ore,NoTypes,Reduces the amount of ore in the cells that the taskforce is occupying.
-29=Begin Production,NoTypes,Forces the owner of the TeamType to auto-produce (start the skirmish AI).
-30=Force Sale,NoTypes,Forces the 'fire sale' of all remaining structures owned by the House to which the TeamType belongs.
-31=Suicide,NoTypes,This instructs the TaskForce to destroy itself. In effect it commits suicide; usually accompanied by an explosion.
-32=Start Weather Storm In...,Number,The parameter is the delay in seconds before triggering a Weather Storm.
-33=End Weather Storm,NoTypes,
-34=Center Map On Team,NoTypes,
-35=Shroud Map For Time Interval,Number,Shrouds the entire map for the duration in frames.
-36=Reveal Map For Time Interval,Number,Reveals the entire map for the duration in frames.
-37=Delete Team Members,NoTypes,Instruct TaskForce members to leave the map after which they are deleted.
-38=Clear Global,Number,Clears a global variable (see Set Global).
-39=Set Local,Number,Sets a local variable (see Clear Local).
-40=Clear Local,Number,Clears a local variable (see Set Local).
-41=Unpanic,NoTypes,Instructs infantry with FraidyCat=yes to calm down.
-42=Change Facing,FacingTypes,
-43=Wait Until Fully Loaded,NoTypes,Transports wait until they are fully loaded before continuing the script.
-44=Unload TRUCKB > TRUCKA,NoTypes,Unloads TRUCKB by changing its type to TRUCKA.
-45=Load TRUCKA > TRUCKB,NoTypes,Loads TRUCKA by changing its type to TRUCKB.
-46=Attack Enemy Structure,BuildingTypes,Instructs the taskforce to attack a specific enemy structure.
-47=Move To Enemy Structure,BuildingTypes,Instructs the taskforce to move to a specific enemy structure.
-48=Scout,NoTypes,Move as far as possible in a random direction. Use this to make the AI scout the map.
-49=Register Success,NoTypes,Register success for AITrigger weight adjustment. Executing this script just once makes its AITrigger successful - used in success/failure delta calculations.
-50=Flash,Number,Flashes the taskforce a number of times.
-51=Play Animation,Number,Plays an animation from [Animations] in RULES.ini.
-52=Display Talk Bubble,TalkBubbleTypes,Displays a talk bubble over the taskforce.
-53=Gather At Enemy Base,NoTypes,
-54=Regroup At Friendly Base,NoTypes,
-55=Activate Iron Curtain,NoTypes,Activates the Iron Curtain on the taskforce if it is fully charged.
-56=Chronosphere TaskForce,BuildingTypes,Chronospheres the taskforce to a specific structure if it is almost fully charged.
-57=Chronowarps TaskForce,BuildingTypes,Chronowarps the taskforce to a specific structure if it fully charged.
-58=Move To Friendly Structure,BuildingTypes,
+; Configuration for RA2.

```

Comment:
Could you fix the line endings so that it doesn't look like the entire file was changed when you only added one or two lines?

Reply:
Updated.